src/main/flight/failsafe.h

```diff
@@ -40,8 +40,9 @@ typedef struct failsafeConfig_s {
     int16_t failsafe_fw_roll_angle;         // Settings to be applies during "LAND" procedure on a fixed-wing
     int16_t failsafe_fw_pitch_angle;
     int16_t failsafe_fw_yaw_rate;
-
     uint16_t failsafe_stick_motion_threshold;
+    uint16_t failsafe_min_distance;              // Minimum distance required for failsafe procedure to be taken. 1 step = 1 meter. 0 = Regluar failsafe_procedure always active (default)        
```

Comment:
Just realised that this in meters where pretty much any other distance is in centimeters. Shouldn't we use centimeters here too? Also, small typo in `Regluar`.

Reply:
I'd use centimeters for consistency with other distance settings.